tests/client/virtual_devices/virtual_device_util.c

```diff
@@ -171,6 +170,10 @@ void lowcar_protocol(int fd, uint8_t type, uint8_t year, uint64_t uid,
 
                 case DEVICE_WRITE:
                     device_write(type, incoming_msg, params);
+                    // If we're writing a pitch to the SoundDevice, play the pitch
+                    if (type == device_name_to_type("SoundDevice")) {
```

Comment:
maybe another approach is to have action_interval = 0 mean that you do the action only once, so that this can be a general option.

Reply:
What I'm trying to achieve here is to call the `device_actions()` every time the device is written to and not call it periodically. All other devices (at the moment) call `device_actions()` periodically and not when a param is written to